torch/_dynamo/source.py

```diff
@@ -617,3 +617,7 @@ def is_from_defaults(source: Source):
     if isinstance(source, ChainedSource):
         return is_from_defaults(source.base)
     return False
+
+
+def is_cell_contents(source: Source):
```

Comment:
what is this doing out of curiosity?

Reply:
This is part of heuristic rules that determinate if we should wrap int as symint. Here we are saying if the value is from a cell closures, we would not make it dynamic since cell closures are usually constant. We define these heuristic based on ```source```.